torch/csrc/jit/script/compiler.h

```diff
@@ -134,6 +149,7 @@ TORCH_API void defineMethodsInModule(
 // same as above but parse the definitions from source
 TORCH_API void defineMethodsInModule(Module & m, const std::string& source, const Resolver& resolver, std::shared_ptr<SugaredValue> self);
 TORCH_API std::shared_ptr<Graph> compileFunction(Def def, const Resolver& resolver);
+TORCH_API std::shared_ptr<Graph> compileFunction(const std::string& source);
```

Comment:
Well a reason to call it `compileFunction` instead of `compile` is that it can only handle a single function, and it will return a graph instead of a runnable thing.